### DISCUSSION TITLE
Is it possible to get Copilot Business to our organisation?

### DISCUSSION THREAD

**Discussion by parland** [arcada-uas/arcada-uas.github.io#1] [Q&A]
<img width="474" alt="image" src="https://github.com/arcada-uas/arcada-uas.github.io/assets/1327504/85b0d065-ec61-4055-941f-e8d23e701304"> <BR/>
If you pay for yourself...

**Comment by thesourmango** [comment]
How does this work, can you send me a link? So copilot is so vague, is it os-level windows only integration, is it AI assistant in Edge, is it AI integration on Github or is it AI chatbot plugin for VSCode, I don't understand :)

**Reply by parland** [reply]
"Copilot" refer to several different AI tools and integrations.

1. **GitHub Copilot**: This is an AI-powered code completion tool integrated into VSCode. It helps developers by suggesting entire lines or blocks of code as they type, based on the context of the existing code and comments. It's specifically designed to assist with coding tasks, leveraging the power of OpenAI's Codex model.

2. **Windows Copilot**: This is a relatively new feature announced by Microsoft, aiming to integrate AI assistance directly into the Windows operating system. It is designed to provide a seamless AI experience across the OS, helping with tasks, suggesting actions, and improving productivity within the Windows environment.

3. **Microsoft Edge Copilot**: This AI assistant is integrated into the Microsoft Edge browser. It helps users by providing contextual information, summarizing web pages, answering queries, and performing other browser-related tasks to enhance the browsing experience.

4. **AI Integration in GitHub**: Apart from GitHub Copilot for VSCode, GitHub is also exploring other AI integrations to help developers with code reviews, issue management, and more, leveraging AI to streamline the software development lifecycle.

Each of these "Copilot" operates within its respective ecosystem. 

Here are some links:

- [GitHub Copilot](https://github.com/features/copilot)
- [Windows Copilot](https://blogs.windows.com/windowsexperience/2023/05/23/introducing-windows-copilot-your-everyday-ai-companion/)
- [Microsoft Edge Copilot](https://www.microsoft.com/en-us/edge)
